sdk/python/go_build.py

```diff
@@ -1,37 +0,0 @@
-# Copyright 2022 The Feast Authors
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     https://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-import os
-import pathlib
-import shutil
-import subprocess
-
-# Build go server for 3 targets: macos (intel), macos (m1), linux (64 bit)
-# First start by clearing the necessary directory
-binaries_path = (pathlib.Path(__file__) / "../feast/binaries").resolve()
```

Comment:
should we delete this code right now? I suspect we'll need a modified version of this logic for the release process of building binaries

Reply:
fine with deleting